modules/fbs/includes/fbs.debt.inc

```diff
@@ -12,11 +12,15 @@
  *   User to get debt listing for.
  * @param bool $reset
  *   If TRUE cache is reset (default: FALSE).
+ * @param int $patron_id
+ *   Id of the patron at the provider.
+ * @param bool $include_paid
+ *   Default to FALSE, if TRUE already paid are also returned.
  *
  * @return array
  *   Debt information.
  */
-function fbs_debt_list($account, $reset = FALSE) {
+function fbs_debt_list($account, $reset = FALSE, $patron_id = NULL, $include_paid = FALSE) {
```

Comment:
I am not very fond of the `$patron_id` being added to the provider signature. We already have the `$account` variable and that should be used if possible. I know that might be hard though and we might not be in a situation were we are interested in changing that.

Reply:
The same goes for `fbs_debt_payment_received()`.

Reply:
It's do the the "user have logged out" and payments are retired at cron runs. So we need the patron id for this to work.

Reply:
@cableman Have you considered alternatives like:

1. Calling FBS directly from `_ding_debt_easy_retry_payments()`? This could create a hard dependency between `fbs` and `ding_debt_easy` (at least for async processing). I would not consider that a dealbreaker.
2. Creating new provider hooks like `ding_provider_invoke('debt', 'list_async')`
3. Loading the proper account based on data stored with the payment so it can be passed to the provider as expected

Reply:
I only see 2 as an option, but I can't see that that would not also be a change to the provider. I don't see 3 will work as there will be a periode where some vil use easy and some will use dibs in a short periode.